lib/handler/general.dart

```diff
@@ -19,6 +22,11 @@ int getLatestSpecVersion(network) {
 }
 
 Future<dynamic> handleRequest(Request request) async {
+  Logger.root.level = Level.ALL;
+  Logger.root.onRecord.listen((record) {
+    print('${record.time} [${record.level.name}] ${record.message}');
+  });
```

Comment:
Wouldn't it need to be defined outside the handleRequest?

Reply:
Can´t do that outside of a function

Reply:
Done